packages/features/troubleshooter/components/TroubleshooterSidebar.tsx

```diff
@@ -29,7 +29,7 @@ export const TroubleshooterSidebar = () => {
   const { t } = useLocale();
 
   return (
-    <div className="relative z-10 hidden w-full flex-col gap-6 h-screen overflow-y-auto py-6 pl-4 pr-6 sm:flex md:pl-0">
+    <div className="relative z-10 hidden h-screen w-full flex-col gap-6 overflow-y-auto py-6 pl-4 pr-6 sm:flex md:pl-0">
```

Comment:
did you add anything here?

Reply:
can you undo this

Reply:
no i just changed it according to prettier.

Reply:
The classes were reshuffled based on prettier

Reply:
> can you undo this? I don't see any change

yes